### DISCUSSION TITLE
Type 'NextServer' is missing

### DISCUSSION THREAD

**Discussion by luiznasciment0** [vercel/next.js#24238] [Help] [answered]
Hey! I'm using Next.js 9.4.0 and I'm trying to update it to the latest version. 
We have some custom server configurations and now it's types are misleading. 

This is the error I'm getting: ```Type 'NextServer' is missing the following properties from type 'Server': dir, quiet, nextConfig, distDir, and 35 more.ts(2740)```
![Screen Shot 2021-04-19 at 17 19 20](https://user-images.githubusercontent.com/55008532/115298090-61bb4900-a133-11eb-8acb-f4e138aa83b5.png)

The type "Server" is: import Server from 'next/dist/next-server/server/next-server'

Can someone help me?

**Comment by KaiohFernandes** [comment]
Up

**Comment by luiznasciment0** [comment, marked as answer]
Just fixed it. I had to replace the ```import Server from 'next/dist/next-server/server/next-server'``` to ```import { NextServer } from 'next/dist/server/next'```.
